.github/scripts/examples_validation.py

```diff
@@ -0,0 +1,44 @@
+import os
+import ast
+import docutils.nodes
+from docutils.core import publish_doctree
+
+# This script is used to validate the examples in the repo. It will validate that the names of the examples are
+# correct and match the name of the check. In addition it will validate the examples include a single H1 tag.
+
+checks_dirs = ["deepchecks/tabular/checks", "deepchecks/vision/checks"]
+
+ignored_files = [
+    "deepchecks/vision/checks/distribution/abstract_property_outliers.py",
+    ]
+
+
+def validate_dir(checks_path, examples_path):
+    for root, subdirs, files in os.walk(checks_path):
```

Comment:
Not sure this is the best way, why not something like
```
from deepchecks.tabular import checks as tabular_checks

tabular_checks.__dict__
```

Reply:
why? I don't care about the module. only about the file path.

Reply:
but you start having ignore_files and stuff instead of just importing the checks you want to validate